.ci/build_wheel.py

```diff
@@ -0,0 +1,94 @@
+# This script generates the different versions of the ansys-dpf-core wheels based on a given input.
+# Input can be one of ["any", "win", "manylinux1", "manylinux_2_17"]
+
+import argparse
+import pathlib
+import subprocess
+import os
+import sys
+import shutil
+import tempfile
+
+
+supported_platforms = {
+    "any": "any",
+    "win": "win_amd64",
+    "manylinux1": "manylinux1_x86_64",
+    "manylinux_2_17": "manylinux_2_17_x86_64"
+}
+
+argParser = argparse.ArgumentParser()
+argParser.add_argument("-p", "--platform", help="platform")
+argParser.add_argument("-w", "--wheelhouse", help="platform", action='store_true')
+
+args = argParser.parse_args()
+
+if args.platform not in supported_platforms:
+    raise ValueError(f"Platform {args.platform} is not supported. "
+                     f"Supported platforms are: {list(supported_platforms.keys())}")
+else:
+    requested_platform = supported_platforms[args.platform]
+print(requested_platform)
+
+# Move binaries out of the source depending on platform requested
+# any: move all binaries out before building
+# win: move .so binaries out before building
+# lin: move .dll binaries out before building
+with tempfile.TemporaryDirectory() as tmpdirname:
+    print('Created temporary directory: ', tmpdirname)
+
+    # Create the temporary build-opts.cfg
+    build_opts_path = os.path.join(tmpdirname, "build-opts.cfg")
+    with open(build_opts_path, "w") as build_opts_file:
+        build_opts_file.write(f"[bdist_wheel]\nplat-name={requested_platform}")
+    os.environ["DIST_EXTRA_CONFIG"] = build_opts_path
+
+    # Move the binaries
+    gatebin_folder_path = os.path.join(
+        os.path.curdir,
+        os.path.join("src", "ansys", "dpf", "gatebin")
+    )
+    binaries_to_move = []
+    moved = []
+    if "win" in requested_platform or "any" == requested_platform:
+        # Move linux binaries
+        binaries_to_move.extend(["libAns.Dpf.GrpcClient.so", "libDPFClientAPI.so"])
+    if "linux" in requested_platform or "any" == requested_platform:
+        # Move windows binaries
+        binaries_to_move.extend(["Ans.Dpf.GrpcClient.dll", "DPFClientAPI.dll"])
+    if "any" == requested_platform:
+        binaries_to_move.extend(["_version.py"])
```

Comment:
@PProfizi _version.py is not needed for the other packages ?

Reply:
@anslpa to make the `any` wheel, I need to completely remove the `gatebin` folder, including this file.
The tests do pass when using this `any` wheel.
I even think we should remove those `_version.py` files from `ansys.grpc.dpf`, `ansys.dpf.gatebin` and `ansys.dpf.gate` altogether.